packages/client/src/views/PrintCertificate/PDFUtils.ts

```diff
@@ -253,9 +266,30 @@ export function svgToPdfTemplate(svg: string, offlineResource: IOfflineData) {
     }
   }
 
-  pdfTemplate.definition.content = {
-    svg
+  const foreignObjects = svgElement.getElementsByTagName('foreignObject')
+  const absolutelyPositionedHTMLs: Content[] = []
+  if (foreignObjects.length) {
```

Comment:
You can drop this if as if it's an empty array, then the loop just runs 0 times :)

Reply:
done